src/tests/work-item/work-item-list/work-item-list.spec.js

```diff
@@ -65,7 +65,7 @@ describe('Work item list', function () {
       expect(page.workItemDescription(page.workItemByNumber(0))).toBe('Some Description');
       expect(page.workItemTitle(page.workItemByNumber(0))).toBe('Some Title');
       page.allWorkItems.getText().then(function (text) {
-        expect(text).toContain("View Details Delete\nnew\n1\nSome Title\nSome Description");
+        expect(text).toContain("new\n1\nSome Title 1\nSome Description 1\nView Details Delete");
```

Comment:
Looks good! Please commit!

Yes - the test code was pretty brittle - sorry that it caused a problem! It was actually a work-around to how the UI included the quick add web elements as the first "row" in the display of work items. Is this how we will ensure that the quick add web elements are always displayed at the "top" of the page? Thx!